<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,3">
    <item>
     <widget class="QStackedWidget" name="leftStackedWidget">
      <widget class="QWidget" name="leftStackedWidgetPage1">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="rightStackedWidget">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="horizontalStackedWidgetPage1_2">
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionRegistration"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Algorithom(&amp;A)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRender_R">
    <property name="title">
     <string>Render(&amp;R)</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuRender_R"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>快捷栏1</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport_Ply"/>
   <addaction name="actionSave_Ply"/>
   <addaction name="actionSave_Skel"/>
   <addaction name="actionSnapShot"/>
   <addaction name="actionDenoising"/>
   <addaction name="actionDown_Sample"/>
   <addaction name="actionSkeleton"/>
   <addaction name="actionEAR"/>
   <addaction name="actionSetting"/>
   <addaction name="actionNormal"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>快捷栏2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actiontemp"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Original"/>
   <addaction name="actionShow_Samples"/>
   <addaction name="actionShow_Skeleton"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Raidus"/>
   <addaction name="actionShow_Normals"/>
   <addaction name="actionCull_Points"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionShow_Original">
   <property name="text">
    <string>Show Original</string>
   </property>
  </action>
  <action name="actionShow_Samples">
   <property name="text">
    <string>Show Samples</string>
   </property>
  </action>
  <action name="actionShow_Skeleton">
   <property name="text">
    <string>Show_Skeleton</string>
   </property>
  </action>
  <action name="actionShow_Raidus">
   <property name="text">
    <string>Show Raidus</string>
   </property>
  </action>
  <action name="actionShow_Normals">
   <property name="text">
    <string>Show Normals</string>
   </property>
  </action>
  <action name="actionCull_Points">
   <property name="text">
    <string>Cull Points</string>
   </property>
  </action>
  <action name="actiontemp">
   <property name="text">
    <string>temp</string>
   </property>
  </action>
  <action name="actionImport_Ply">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/import.png</normaloff>:/import.png</iconset>
   </property>
   <property name="text">
    <string>Import Ply</string>
   </property>
   <property name="toolTip">
    <string>Import Ply</string>
   </property>
  </action>
  <action name="actionSave_Ply">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/图标/云存储.png</normaloff>:/图标/云存储.png</iconset>
   </property>
   <property name="text">
    <string>Save Ply</string>
   </property>
  </action>
  <action name="actionSave_Skel">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/图标/文件.png</normaloff>:/图标/文件.png</iconset>
   </property>
   <property name="text">
    <string>Save Skel</string>
   </property>
  </action>
  <action name="actionSnapShot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/图标/风景.png</normaloff>:/图标/风景.png</iconset>
   </property>
   <property name="text">
    <string>SnapShot</string>
   </property>
  </action>
  <action name="actionDenoising">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/图标/清除.png</normaloff>:/图标/清除.png</iconset>
   </property>
   <property name="text">
    <string>Denoising</string>
   </property>
  </action>
  <action name="actionDown_Sample">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/图标/相机转换.png</normaloff>:/图标/相机转换.png</iconset>
   </property>
   <property name="text">
    <string>Down Sample</string>
   </property>
  </action>
  <action name="actionSkeleton">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/图标/箭头.png</normaloff>:/图标/箭头.png</iconset>
   </property>
   <property name="text">
    <string>Skeleton</string>
   </property>
  </action>
  <action name="actionEAR">
   <property name="text">
    <string>EAR</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/图标/应用中心.png</normaloff>:/图标/应用中心.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/图标/工作台.png</normaloff>:/图标/工作台.png</iconset>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRegistration">
   <property name="text">
    <string>Registration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
